integration-tests/src/test/java/oracle/weblogic/kubernetes/ItJrfMiiDomain.java

```diff
@@ -0,0 +1,327 @@
+    logger.info("EM console is accessible thru default service");
+  }
+
+  /**
+   * Construct a domain object with the given parameters that can be used to create a domain resource.
+   */
+  private Domain createDomainResource(
+      String domainUid, String domNamespace, String adminSecretName,
+      String repoSecretName, String encryptionSecretName, String rcuAccessSecretName,
+      String opssWalletPasswordSecretName, int replicaCount, String miiImage) {
+    // create the domain CR
+    Domain domain = new Domain()
+            .apiVersion(DOMAIN_API_VERSION)
+            .kind("Domain")
+            .metadata(new V1ObjectMeta()
+                    .name(domainUid)
+                    .namespace(domNamespace))
+            .spec(new DomainSpec()
+                    .domainUid(domainUid)
+                    .domainHomeSourceType("FromModel")
+                    .image(miiImage)
+                    .imagePullPolicy("IfNotPresent")
+                    .addImagePullSecretsItem(new V1LocalObjectReference()
+                            .name(repoSecretName))
+                    .webLogicCredentialsSecret(new V1SecretReference()
+                            .name(adminSecretName)
+                            .namespace(domNamespace))
+                    .includeServerOutInPodLog(true)
+                    .serverStartPolicy("IF_NEEDED")
+                    .serverPod(new ServerPod()
+                            .addEnvItem(new V1EnvVar()
+                                    .name("JAVA_OPTIONS")
+                                    .value("-Dweblogic.StdoutDebugEnabled=false"))
+                            .addEnvItem(new V1EnvVar()
+                                    .name("USER_MEM_ARGS")
+                                    .value("-Djava.security.egd=file:/dev/./urandom ")))
+                    .adminServer(new AdminServer()
+                            .serverStartState("RUNNING")
+                            .adminService(new AdminService()
+                                    .addChannelsItem(new Channel()
+                                            .channelName("default")
+                                            .nodePort(0))))
+                    .addClustersItem(new Cluster()
+                            .clusterName("cluster-1")
+                            .replicas(replicaCount)
+                            .serverStartState("RUNNING"))
+                    .configuration(new Configuration()
+                            .opss(new Opss()
+                                   .walletPasswordSecret(opssWalletPasswordSecretName))
+                            .model(new Model()
+                                    .domainType("JRF")
+                                    .runtimeEncryptionSecret(encryptionSecretName))
+                            .addSecretsItem(rcuAccessSecretName)
+                            .introspectorJobActiveDeadlineSeconds(600L)));
+
+    return domain;
+  }
+
+  private void checkAppIsRunning(
```

Comment:
doesn't look like you are calling this method, delete it

Reply:
Yeah, for this JIRA's work I don't need it. Let me remove it for now.

Reply:
Done